specs/RuntimeSpec/Fees/Fees.md

```diff
@@ -73,7 +68,40 @@ Here is the list of actions and their corresponding fees:
     - the base fee [`delete_key_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#delete_key_cost)
 - [DeleteAccount](/RuntimeSpec/Actions.md#deleteaccountaction) uses
     - the base fee [`delete_account_cost`](/GenesisConfig/RuntimeFeeConfig/ActionCreationConfig.md#delete_account_cost)
-
+    - action receipt creation fee for creating Transfer to send remaining funds to `beneficiary_id`
+    - full transfer fee described in the corresponding item
+    
+## Gas tracking
+
+In `Runtime`, gas is tracked in the following fields of `ActionResult` struct:
+- `gas_burnt` - irreversible amount of gas spent on computations.
+- `gas_used` - includes burnt gas and gas attached to the new `ActionReceipt`s created during the method execution.
+- `gas_burnt_for_function_call` - stores gas burnt during function call execution. Later, owner of the corresponding contract gets 30% of it as a reward for a possibility to invoke the function.
+
+Initially runtime charges all used gas from the account. But during refund computation, if new `ActionReceipt`s are not actually going to be sent, e.g. in case of failure on applying current receipt, difference between used gas and burnt gas is returned to the account.
```

Comment:
I think this is still misleading. You can attach more gas than needed, which will cause `gas_used` to be higher than `gas_burnt`